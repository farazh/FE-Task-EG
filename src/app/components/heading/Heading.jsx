import React from "react";
import "./Heading.scss";

export const Heading = () => (
  <div className="heading">
    <h1 className="title">
      <i>Marvelous!</i>
    </h1>
    <div className="text">
      Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
      Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
      Lorem ipsum Lorem ipsum Lorem ipsum
    </div>
  </div>
);
