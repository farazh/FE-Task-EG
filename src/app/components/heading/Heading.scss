$break-point: 760px;

.title {
  color: white;
}

.text {
  color: #00d9d9;
  font-weight: bold;
}

@media (min-width: $break-point) {
  .text {
    padding: 0 50% 0 0;
  }

  .heading {
    text-align: left;
  }
}
