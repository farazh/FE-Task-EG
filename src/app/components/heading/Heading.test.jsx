import React from "react";
import { shallow } from "enzyme";
import { Heading } from "./Heading";

describe("Heading tests", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Heading />);
    expect(wrapper).toMatchSnapshot();
  });
});
