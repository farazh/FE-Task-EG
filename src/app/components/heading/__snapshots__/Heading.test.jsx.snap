// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading tests should match snapshot 1`] = `
<div
  className="heading"
>
  <h1
    className="title"
  >
    <i>
      Marvelous!
    </i>
  </h1>
  <div
    className="text"
  >
    Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
  </div>
</div>
`;
